PlaybackLab/BasicPlaybackWithScan/app/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
 }
 
 task copyTask(type: Copy) {
-    from new File(projectDir, '../../vendor/Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
+    from new File(projectDir, '../../../vendor/Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
```

Comment:
I could do something like this for all projects

IMASampleApp/build.grade
```
ext {
    vendorDir = new File(projectDir, '../vendor')
}
```

IMASampleApp/app/build.grade
```
task copyTask(type: Copy) {
    from new File(vendorDir, 'Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
    from new File(vendorDir, 'Ooyala/OoyalaIMASDK-Android/OoyalaIMASDK.jar')
    from new File(vendorDir, 'Google/ima-android-v371.jar')
    into new File(projectDir, './libs')
    doLast {
        println "Jars updated."
    }
}
```

Do you prefer it?